src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -120,79 +80,63 @@ const FormattedReport = ({ summaries, weekIndex }) => {
-          toastId: toastGoogleLinkDoesNotExist,
-          pauseOnFocusLoss: false,
-          autoClose: 3000,
-        },
-      );
-    }
-  };
-
-  const handleChangeBioPosted = async (userId, bioStatus) => {
-    try {
-      const url = ENDPOINTS.USER_PROFILE(userId);
-      const response = await axios.get(url);
-      const userProfile = response.data;
-      const res = await axios.put(url, {
-        ...userProfile,
-        bioPosted: !bioStatus,
-      });
-      if (res.status === 200) {
-        toast.success('You have changed the bio announcement status of this user.');
-      }
-    } catch (err) {
-      alert('An error occurred while attempting to save the bioPosted change to the profile.');
-    }
-  };
-
   return (
     <>
       {alphabetize(summaries).map((summary, index) => {
         const hoursLogged = (summary.totalSeconds[weekIndex] || 0) / 3600;
-        const googleDocLink = getGoogleDocLink(summary);
-        const [isBioPosted, setIsBioPosted] = useState(summary.bioPosted);
+
         return (
           <div
             style={{ padding: '20px 0', marginTop: '5px', borderBottom: '1px solid #DEE2E6' }}
             key={'summary-' + index}
           >
-            <div>
+            <p>
               <b>Name: </b>
               <Link to={`/userProfile/${summary._id}`} title="View Profile">
-                {summary.firstName} {summary.lastName}
+                {summary.firstName} {summary.lastName}{' '}
               </Link>
 
-              <span onClick={() => handleGoogleDocClick(googleDocLink)}>
-                <img className="google-doc-icon" src={google_doc_icon} alt="google_doc" />
-              </span>
-            </div>
-            <div>
+              {hoursLogged > summary.weeklycommittedHours && (
+                <i
+                  className="fa fa-star"
+                  title={`Weekly Committed: ${summary.weeklycommittedHours} hours`}
+                  style={{
+                    color:
+                      hoursLogged >= summary.weeklycommittedHours * 1.75
+                        ? 'purple'
+                        : hoursLogged >= summary.weeklycommittedHours * 1.5 &&
+                          hoursLogged < summary.weeklycommittedHours * 1.75
+                        ? 'fuchsia'
+                        : hoursLogged >= summary.weeklycommittedHours * 1.25 &&
+                          hoursLogged < summary.weeklycommittedHours * 1.5
+                        ? 'lightgreen'
+                        : 'green',
+                    fontSize: '55px',
+                    margin: 'auto',
+                    verticalAlign: 'middle',
+                    position: 'relative',
+                  }}
+                >
+                  <span
+                    style={{
+                      position: 'absolute',
+                      top: '50%',
+                      left: '50%',
+                      transform: 'translate(-50%, -50%)',
+                      color: 'black',
```

Comment:
How about changing this color to white?
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/44cb22f1-0ff2-4a9d-ac9c-834e23083f6f)

Reply:
Thank you for the suggestion @eduhorta . However the white color was not readable on light green background.

Reply:
I tried changing the color  of the text to white and this is how it looks:
![PR#806-white text](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/1cc94594-a84e-4d4d-abf3-350d3ba9826d)
In this situation maybe we can saturate the green color so that the white text is more visible (if white text is finalised) or maybe we can consider a different color that contrasts white. For this, maybe referring this link can help https://color.adobe.com/create/color-wheel
If no other color is found, then maybe we can try exploring outlining the text inside the star.